src/app/log-in/log-in.component.html

```diff
@@ -2,7 +2,7 @@
 
 <div
   class="container home-container text-center"
-  *ngIf="globalVars.inTab || globalVars.webview"
+  *ngIf="globalVars.inTab || globalVars.webview || globalVars.redirectURI"
```

Comment:
is there a reason we don't have `|| globalVars.callback` in here?

Reply:
So my theory is that callback is really only used for mobile app use cases, and I believe mobile apps pass the webview param as well, so for the purpose of some of these template conditions I *think* they are interchangeable.